src/errors/test-run/templates.js

```diff
@@ -187,9 +187,15 @@ export default {
     `),
 
     [TYPE.actionIncorrectKeysError]: err => markup(err, `
-         The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
+        The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
 
-         ${err.getCallsiteMarkup()}
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.hybridFnExecutingInterruptedError]: err => markup(err, `
+        Client code execution was interrupted by page unload.
```

Comment:
More likely the reason will be changing triger of page navigation from Hybrid code. This is what we should suggest, smthg like: `This problem may appear if you trigger page navigation from client code.`